JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -61,5 +66,11 @@ class ViewController: UIViewController {
         updateStock()
```

Comment:
처음에는 catch JuiceMakerError.outOfStock { } 만 적용할 경우
'Errors thrown from here are not handled because the enclosing catch is not exhaustive'
에러가 발생해서 NSError 를 처리해주기 위해서 catch를 한번 더 사용했었습니다.

그래서 이번에는 리뷰해주신대로 catch 하나로 처리해보았습니다만, 이렇게 진행할 경우 JuiceMakerError와 NSError가 구분이 안될 것 같은데 이대로 진행해도 괜찮은지 궁금해서 질문 드립니다 🤔
이번 프로젝트에서는 NSError가 발생할 일이 없긴 하겠지만, 모든 에러를 처리해주지 않고 구분없이 하나의 catch문으로 묶어서 처리해도 괜찮은 것일까요?

Reply:
여기서는 에러를 하나만 던져주는게 확실해서 catch 하나로만 처리하는게 깔끔해보여요!

Catch 안에서 분기처리로 stockError인지 한번 검사하는 것도 좋을 것 같아요 :)

Reply:
넵 말씀해주신대로 catch 구문 안에서 stockError인지 검사해주는 방식으로 처리했습니다 ☺️
감사합니다 🙏